recipes/onetbb/all/conanfile.py

```diff
@@ -143,6 +145,8 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
```

Comment:
```suggestion
        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
```

there is no pdb files in lib folder from what I see in previous logs.